packages/fxa-auth-client/lib/crypto.ts

```diff
@@ -206,6 +206,39 @@ export async function jweEncrypt(
   return compactJWE;
 }
 
+/**
+ * Decrypt a JWE token using provided key material
+ * @param keyMaterial Key material
+ * @param jwe JWE token
+ */
+export async function jweDecrypt(keyMaterial: Uint8Array, jwe: string) : Promise<string> {
+ const encoder = new TextEncoder();
```

Comment:
Indentation is off?  (no auto-formatting?)

Reply:
Filed https://mozilla-hub.atlassian.net/browse/FXA-7109